test/modal-test.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import { noop } from 'lodash';
+
+import { Modal } from '../src';
+
+describe('<Modal/>', function() {
+  describe('With default properties', function() {
+    const component = shallow(<Modal onClose={noop}/>);
+    it('Should have a size of medium', function() {
+      expect(component.find('div.modal').hasClass('medium-panel')).to.equal(true);
+    });
+    it('Should be aligned left', function() {
+      expect(component.find('div.modal-body').hasClass('')).to.equal(true);
```

Comment:
`.hasClass('')` will always evaluate to true - if the expected behavior is that no class will be present, we can't test for that with this.  We could test `expect(component.find('div.modal-body').props().className).to.equal('the other default classes');` but I don't think that is necessary.  I think this can be removed.